packages/perseus/src/widgets/cs-program/cs-program.tsx

```diff
@@ -95,6 +96,18 @@ class CSProgram extends React.Component<Props> implements Widget {
         return _getPromptJSON();
     }
 
+    /**
+     * @deprecated and likely very broken API
+     * [LEMS-3185] do not trust serializedState
+     */
+    getSerializedState(): any {
```

Comment:
I added/updated `getSerializedState` in a couple of places to keep the serialized state shape consistent.